src/main/scala/apps/femur/PosteriorVariabilityToMeshColor.scala

```diff
@@ -31,15 +32,15 @@ object PosteriorVariabilityToMeshColor {
 
     val logPath = new File(dataFemurPath, "log")
 
-    val (model, _, _, _) = LoadTestData.modelAndTarget()
+    val (model, _, targetMesh, _) = LoadTestData.modelAndTarget()
 
     val jsonFileName = "icpProposalRegistration.json"
 
     println(new File(logPath, jsonFileName).toString)
 
     val logObj = new JSONAcceptRejectLogger[ModelFittingParameters](new File(logPath, jsonFileName))
     val logInit: IndexedSeq[jsonLogFormat] = logObj.loadLog()
-    val burnInPhase = 1000
+    val burnInPhase = 300
```

Comment:
Optional improvement. Parameters could be put into a config file instead of having them hardcoded